gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java

```diff
@@ -223,8 +223,10 @@ public void onAddSpec(Spec addedSpec) {
           _log.info("Scheduling flow spec: " + addedSpec);
           scheduleJob(jobConfig, null);
           if (jobConfig.containsKey(ConfigurationKeys.FLOW_RUN_IMMEDIATELY)) {
```

Comment:
This will be cleaner with fewer indentation levels as `if (PropertiesUtils.getPropAsBoolean(jobConfig, ConfigurationKeys.FLOW_RUN_IMMEDIATELY, "false"))`

Reply:
Thanks for the suggestion! Incorporated this change.